src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -66,8 +66,19 @@
                               :onClick #(swap! state update :expanded not)})))
         [:div {:style {:padding "0.25em 0 0 0"} :id "chart_div"}]
         (if (:expanded @state)
-          (.timingDiagram js/window (:data props) (:workflow-name props))
-          (when (.getElementById js/document "chart_div") (gdom/remove-children "chart_div")))])})
+          (.timingDiagram js/window (:data props) (:workflow-name props) nil)
+          (when (.getElementById js/document "chart_div") (gdom/remove-children "chart_div")))])
+   :component-did-update
+   (fn [{:keys [props state]}]
+     (if (:expanded @state)
+       (js/setTimeout
+         (fn []
+           (let
+             [height (aget (aget (aget (aget (.-childNodes (aget (.-childNodes (aget (.-childNodes (aget (.-childNodes (aget
```

Comment:
This line throws an error for me: `Uncaught TypeError: Cannot read property 'childNodes' of undefined`.